### PR TITLE
RegisterForm component created, LoginPage created

### PR DESCRIPTION
**Pull request by Maria-Chira**
RegisterForm component

## Description

RegisterForm created and LoginPage created

## Related to

Ticket: RegisterForm component

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have carefully reviewed my own code
- [x] I have commented my code
- [ ] I have updated any documentation

### CODE REVIEW

File: client/src/App.js [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
 import About from "./pages/About";
-import Home from "./pages/Home";
+import LoginPage from "./pages/LoginPage";
 
 const App = () => (
 	<Routes>
-		<Route path="/" element={<Home />} />
-		<Route path="/about/this/site" element={<About />} />
+		<Route path="/" element={<LoginPage />} />
+		<Route path="/about/this/side" element={<About />} />
```
Comment (TomRafferty) [inline client/src/App.js:9]: did youo mean to put 'site' here?

File: client/src/components/RegisterForm.js [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+import styled from "@emotion/styled";
+import React from "react";
+
+//stores the options for the dropdown lists
+const roles = ["Admin", "Student"];
+const regions = ["London", "West Midlands", "Scotland", "Manchester"];
+const classes = ["1", "2", "3", "4", "5", "6", "7", "8", "9", "10", "11", "12", "13", "14", "15"];
+
+const RegisterContainer = styled(Container)({
+	background: "#fafafa",
```
Comment (TomRafferty) [inline client/src/components/RegisterForm.js:14]: weird tab (wont affect anything though I don't think)

File: client/src/components/RegisterForm.js [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+						margin="dense"
+					/>
+
+					{/* Field for retyping password */}
+					<TextField
+						id="retype-password"
+						label="Retype Password"
+						type="password"
+						margin="dense"
+					/>
```
Comment (TomRafferty) [inline client/src/components/RegisterForm.js:119]: I like that you've added this extra step here :D

**Review by TomRafferty — CHANGES_REQUESTED** [review]
I'm only requesting changes for the small typo at the top (or at least I think it's a typo) the rest of this looks very clean and well commented!

File: client/src/App.js (hunk 1 above)
Comment (Maria-Chira) [reply client/src/App.js:9]: This will be updated by John as discussed. It will be the path for StudentPage

**Review by Johng117 — APPROVED** [review]
Looks good to me.

File: client/src/App.js (hunk 1 above)
Comment (TomRafferty) [reply client/src/App.js:9]: ah no worries then. once John has reviewed the branch can be merged and I'll pull this and fix us up a login component as well as some nice buttons with material UI, I've finished watching a video on it now and I think I get the jist - I also have the docs open in a tab ready too :D

**Review by TomRafferty — APPROVED** [review]
LGTM

**Comment by TomRafferty** [comment]
We are merged - great work Maria!

File: client/src/components/RegisterForm.js [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+    flexDirection: "column",
+    justifyContent: "center",
+});
+const RegisterButton = styled(Button)({
+	marginTop: "10px",
+});
+
+const RegisterForm = () => {
+    //states for the options in the dropdown lists
+    const [role, setRole] =React.useState("");
```
Comment (milleymilli) [inline client/src/components/RegisterForm.js:27]: - Don't you think using **1 useState hook** (in this case) to update multiple values in React is a good practise?
_**I suggest you to try it.**_

File: client/src/components/RegisterForm.js [hunk 5]
```diff
@@ -0,0 +1,129 @@
...
+    const [region, setRegion] = React.useState("");
+    const [classNr, setClassNr] = React.useState("");
+
+    //events for updating the value of the selected options
+    const handleChangeRole = (event) => {
```
Comment (milleymilli) [inline client/src/components/RegisterForm.js:32]: - You may end up with a single event handler function  if you use one **1 useState.** hook.
Which is great!!

Comment (Maria-Chira) [reply client/src/components/RegisterForm.js:32]: All this changes have been added in the last PR

File: client/src/components/RegisterForm.js (hunk 4 above)
Comment (Maria-Chira) [reply client/src/components/RegisterForm.js:27]: All this changes have been added in the last PR

**Pull request merged**